rel/overlay/etc/default.ini

```diff
@@ -578,3 +584,44 @@ compaction = false
 ; CouchDB will use the value of `max_limit` instead. If neither is
 ; defined, the default is 2000 as stated here.
 ; max_limit_partitions = 2000
+
+[reshard]
+; Maximum resharding jobs allowed. New jobs cannot be added until some
+; old ones are deleted. A finished job is not automatically
+; removed. It has to be removed by hand.
```

Comment:
This seems a pretty important distinction from replication, and it's not present in the docs repo:

https://github.com/apache/couchdb-documentation/blob/a35ea115a71fd27a21245a6f14becf43adcfc8a7/src/config/resharding.rst

Nor is it here:

https://github.com/apache/couchdb-documentation/blob/a35ea115a71fd27a21245a6f14becf43adcfc8a7/src/cluster/sharding.rst

Can you add it there please? Thanks.

Reply:
Good catch, I will update it

Reply:
`max_jobs` effect is mentioned in the guide

>  The total number of jobs is important to keep an eye on because there is a maximum number of resharding jobs per node, and creating new jobs after the limit has been reached will result in an error. Before staring new jobs it's a good idea to remove already completed jobs. See :ref:`reshard configuration section <config/reshard>` for the default value of max_jobs parameter and how to adjust if needed.